tests/gold_tests/autest-site/trafficserver.test.ext

```diff
@@ -136,14 +138,15 @@ def MakeATSProcess(obj, name, command='traffic_server', select_ports=True):
     p.Variables.LOCALSTATEDIR = runtime_dir
 
     p.Setup.MakeDir(runtime_dir)
+    p.Setup.Chown(runtime_dir, "nobody", "root", ignore=True)
     # will need this for traffic_manager is it runs
     p.Setup.MakeDir(os.path.join(config_dir, 'snapshots'))
     p.Env['PROXY_CONFIG_SNAPSHOT_DIR'] = os.path.join(config_dir, 'snapshots')
     ##########################################################
     # create subdirectories that need to exist (but are empty)
     # ssl directory has to be created for keeping certs and keys
     p.Setup.MakeDir(ssl_dir)
-    p.Setup.Chown(ssl_dir, "nobody", "nobody", ignore=True)
+    p.Setup.Chown(ssl_dir, "nobody", "root", ignore=True)
```

Comment:
After looking at this in detail.. this can cause pain as the user will have to use sudo to remove the sandbox

to get around this (as we have to update the API for Setup.Chown to have optional group or owner inputs) do the Chown twice with ignore=True as such:


```
p.Setup.Chown(ssl_dir, "nobody", "nogroup", ignore=True) # covers ubuntu
p.Setup.Chown(ssl_dir, "nobody", "nobody", ignore=True)  # cover everyone else
```